### DISCUSSION TITLE
Possible adoption of rioxarray by TorchGeo

### DISCUSSION THREAD

**Discussion by adamjstewart** [corteva/rioxarray#839] [Ideas]
Hi @snowman2,

My name is Adam Stewart and I am the developer of [TorchGeo](https://github.com/microsoft/torchgeo), a PyTorch domain library for working with geospatial data (especially satellite imagery). TorchGeo has long relied on [R-tree](https://github.com/Toblerity/rtree) for spatiotemporal indexing, but it has limited many of the time-series features we would like to add. We are actively exploring other alternatives, including rioxarray. I've been perusing the documentation, but would love to personally meet with you over Zoom if you have time to discuss which features rioxarray does and does not support (or that we could potentially contribute). I'm pretty free for the rest of this month (in the US) and next month (in Germany) if you would like to propose a few times that work for you. Looking forward to speaking with you!

Adam Stewart (my email is on my GitHub profile)

**Comment by adamjstewart** [comment]
Same comment about geocube, since you're also the primary developer of that.

**Comment by snowman2** [comment]
#588 is an issue you may be interested in following.
